Description: The help files have been moved outside of the python module, adjust the path to them
Author: Laurent Bigonville <bigon@debian.org>
Forwarded: not-needed

--- a/sepolicy/sepolicy/gui.py
+++ b/sepolicy/sepolicy/gui.py
@@ -95,6 +95,8 @@ SYSTEM_PAGE = 7
 FILE_EQUIV_PAGE = 8
 START_PAGE = 9
 
+SEPOLICY_SHARED_FILE_PATH = "/usr/share/sepolicy/"
+
 keys = ["boolean", "fcontext", "fcontext-equiv", "port", "login", "user", "module", "node", "interface"]
 
 DISABLED_TEXT = _("""<small>
@@ -129,7 +131,7 @@ class SELinuxGui():
         self.filter_txt = ""
         builder = Gtk.Builder()  # BUILDER OBJ
         self.code_path = distutils.sysconfig.get_python_lib(plat_specific=True) + "/sepolicy/"
-        glade_file = self.code_path + "sepolicy.glade"
+        glade_file = SEPOLICY_SHARED_FILE_PATH + "sepolicy.glade"
         builder.add_from_file(glade_file)
         self.outer_notebook = builder.get_object("outer_notebook")
         self.window = builder.get_object("SELinux_window")
@@ -721,7 +723,7 @@ class SELinuxGui():
         self.back_button.set_sensitive(self.help_page != 0)
         self.forward_button.set_sensitive(self.help_page < (len(self.help_list) - 1))
         try:
-            fd = open("%shelp/%s.txt" % (self.code_path, self.help_list[self.help_page]), "r")
+            fd = open("%shelp/%s.txt" % (SEPOLICY_SHARED_FILE_PATH, self.help_list[self.help_page]), "r")
             buf = fd.read()
             fd.close()
         except IOError:
@@ -729,7 +731,7 @@ class SELinuxGui():
         help_text = self.help_text.get_buffer()
         help_text.set_text(buf % {"APP": self.application})
         self.help_text.set_buffer(help_text)
-        self.help_image.set_from_file("%shelp/%s.png" % (self.code_path, self.help_list[self.help_page]))
+        self.help_image.set_from_file("%shelp/%s.png" % (SEPOLICY_SHARED_FILE_PATH, self.help_list[self.help_page]))
         self.show_popup(self.help_window)
 
     def on_help_back_clicked(self, *args):
